### DISCUSSION TITLE
Question about E2AP Version in srsRAN

### DISCUSSION THREAD

**Discussion by Ali78k** [srsran/srsRAN_Project#900] [General Help] [answered]
Hi srsRAN team,

I'm working with the srsRAN project and encountered some conflicting information in the documentation regarding E2AP version compatibility.

In one section, it specifies that 

> Note that at least Wireshark version 4.0.7 is needed to correctly decode and display E2AP packets (i.e., earlier Wireshark versions do not support E2APv3 protocol and as a result will display information about the Malformed Packets).

However, another section of the documentation includes the following FlexRIC installation command:

```bash
cmake -DKPM_VERSION=KPM_V3 -DXAPP_DB=NONE_XAPP ../
```
In FlexRIC's own documentation, it is stated that if `-DE2AP_VERSION` isn’t explicitly defined, the default E2AP version is set to `V2.03`.

Could you clarify which version of E2AP is actually implemented in srsRAN? Specifically, is it compatible with E2APv3 as the Wireshark version note suggests, or does it default to E2APv2.03 as FlexRIC’s settings imply?

Thank you!

**Comment by pgawlowicz** [comment]
We use asn1 definition from TS document named `O-RAN.WG3.E2GAP-R003-v03.01`.

**Reply by Ali78k** [reply]
@pgawlowicz Thank you for the clarification!

Unfortunately, I couldn’t access the O-RAN.WG3.E2GAP-R003-v03.01 specification. If there’s an accessible link or additional guidance on where to find this specific version, it would be greatly appreciated. Thanks again for your help!

**Reply by pgawlowicz** [reply, marked as answer]
Hi, there was a mistake, it should be `O-RAN.WG3.E2AP-R003-v03.01`.

You can find it [here](https://orandownloadsweb.azurewebsites.net/specifications?QueryOptions=%7B%22SearchString%22%3A%22e2ap%22%2C%22DocumentTypeIds%22%3Anull%2C%22Version%22%3Anull%2C%22WorkGroupId%22%3A13%2C%22ReleaseId%22%3Anull%2C%22FeaturePackageId%22%3Anull%2C%22FileType%22%3Anull%2C%22ShowLatest%22%3Afalse%2C%22IsVisible%22%3Afalse%2C%22PublicationDateRange%22%3A%7B%22StartDate%22%3Anull%2C%22EndDate%22%3Anull%7D%7D)

Be sure to disable the `Show latest` switch.
